lib/nero/blank/blank_algorithm.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'blank_classification'
+require_relative 'blank_subject_state'
+
+module Nero
+  module Blank
+    class BlankAlgorithm < Nero::Algorithm
+      def process(classification, user_state, subject_state)
+        super(classification, user_state, subject_state)
+        return unless classification.user_id
+        return unless classification.subject_ids.size == 1
+
+        classification = Nero::Blank::BlankClassification.new(classification)
+        subject_state = Nero::Blank::BlankSubjectState.new(subject_state)
+        subject_state.add_vote(classification.id, classification.vote(task_key))
+        @storage.record_subject_state(subject_state)
+
+        if subject_state.retired?(blank_consensus_limit) == :blank_consensus
```

Comment:
I wasn't convinced i needed to supply the options to the subject_state initializer but could move this there instead of passing the param here.